lib/pymedphys/_experimental/streamlit/apps/sum_doses.py

```diff
@@ -121,7 +123,9 @@ def patient_ids_in_datasets_are_equal(datasets: list[pydicom.dataset.Dataset]) -
     return all(ds.PatientId == datasets[0].PatientId for ds in datasets)
 
 
-def sum_doses_in_datasets(datasets: list[pydicom.dataset.Dataset]):
+def sum_doses_in_datasets(
+    datasets: Sequence[pydicom.dataset.Dataset],
+) -> pydicom.dataset.Dataset:
```

Comment:
Anytime a top level function has a type hint that utilises an optional dependency, can we surround it in quotes? That way the `pydicom` module isn't accessed at import time, and the dependency is still able to be optional 🙂 


```suggestion
) -> "pydicom.dataset.Dataset":
```

Reply:
I'm happy to do this :smile:. Does this also apply to parameter type hints?

Reply:
> Does this also apply to parameter type hints?

Yup 👍